aggregators/rules/ipg.js

```diff
@@ -1,17 +1,49 @@
+const jsDOM = require("jsdom");
 class IPG {
     constructor() {
-        this.Location = "http://blogs.magicjudges.org/rules/ipg/";
+        this.location = "http://blogs.magicjudges.org/rules/ipg";
     }
 
     find(parameter, callback) {
-        //todo
+        jsDOM.env(
+            this.location + parameter,
+            ["http://code.jquery.com/jquery.js"],
+            /**
+             *
+             * @param err
+             * @param window
+             * @param window.$ jquery
+             * @param window.$.each jquery for each loop
+             * @param window.$.prop jquery property
+             */
+            function(err,window){
+                let resultLength = (window.$(".entry-header h1:first").text()).length+6;
```

Comment:
`resultLength` seems unnecessary, you could just use `result.join("").length` instead.